test/nsidc-icesat2/NSIDC-ICESAT2_Regression.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b26a933e-65ac-4a7a-b298-855c0b845ff1",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# NSIDC ICESat2 Regression tests\n",
+    "\n",
+    "### This juypter notebook runs and verifies a series of test requests against NSIDC's ICESat2 data.\n",
+    "\n",
+    "Requests are submitted and the retrieved data compared to a set of verified results.\n",
+    "\n",
+    "Sample requests include:\n",
+    "\n",
+    "- Subset by bounding box\n",
+    "- Subset by temporal range\n",
+    "- Subset by shapefile \n",
+    "\n",
+    "\n",
+    "We test against ICESat2 v6 collections:\n",
+    "[ATL03](https://nsidc.org/data/atl03/versions/6), [ATL04](https://nsidc.org/data/atl04/versions/6), [ATL06](https://nsidc.org/data/atl06/versions/6), [ATL07](https://nsidc.org/data/atl07/versions/6), [ATL08](https://nsidc.org/data/atl08/versions/6), [ATL09](https://nsidc.org/data/atl09/versions/6), [ATL10](https://nsidc.org/data/atl10/versions/6), [ATL12](https://nsidc.org/data/atl12/versions/6) and [ATL13](https://nsidc.org/data/atl13/versions/6)\n",
+    "\n",
+    "\n",
+    "## Prerequisites\n",
+    "\n",
+    "The dependencies for running this notebook are listed in the\n",
+    "[environment.yaml](https://github.com/nasa/harmony-regression-tests/blob/main/test/nsidc-icesat2/environment.yaml).\n",
+    "\n",
+    "In order to test locally, run the following commands from the `test/nsidc-icesat2/` directory to create and activate the conda environment necessary to run the regression testing notebook.\n",
+    "\n",
+    "```sh\n",
+    "conda env create -f ./environment.yaml && conda activate papermill-nsidc-icesat2\n",
+    "```\n",
+    "\n",
```

Comment:
```suggestion
    "\n",
"To use this environment within a shared Jupyter Hub, see [instructions](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/contributing/workflow.html#create-a-jupyter-kernel-to-run-notebooks) in the NASA Earthdata Cloud Cookbook for how to create a new kernel based on this environment.

Reply:
I tried testing this within the Openscapes 2i2c Jupyter Hub and needed these extra steps to create the kernel. This then worked for me.

Reply:
This is cool, I haven't run anything in openscapes.